readme/public_doc/GLOSSARY/glossary.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Glossary"
+slug: "glossary"
+excerpt: "This section defines key terrarium terms."
+hidden: false
+category: 64fad68e2e32750074872bd0
+---
+
+
+
+# Terrarium Glossary
+
+| Term                               | Definition                                                                                                                         |
+| ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| Terrarium Farm                     | The Terrarium Farm is a repository containing curated collection of content that enables essential features such as generalization of infrastructure dependency interfaces, taxonomy management and auto-complete suggestions for terraform and app-dependencies. |
+| Terrarium Dependency Interface     | The Terrarium Dependency Interface is a contract that defines the inputs and outputs of an infrastructure dependency, facilitating communication between applications and Infrastructure as Code (IaC). |
+| Terrarium Platform Framework       | The Terrarium Platform Framework facilitates the creation of reusable Infrastructure as Code configurations using Terraform. Templates built using this framework are referred to as Terrarium Platform templates. |
+| Terrarium App Metadata             | Terrarium App metadata provides a way for applications to declare their infrastructure dependencies. |
+| CLI                                | Our command-line interface (CLI) offers a seamless experience for working with Terrarium. It includes commands for Terrarium Auto-Complete, leveraging the Terrarium Platform framework, adding dependencies to your applications, and ultimately composing a fully functional Terraform template by combining the Platform template and the app-specific dependencies. |
+| VS Code Extension                  | Our VS Code extension is designed to assists DevOps in writing Terraform code and streamline infrastructure dependency declaration for the app Developers. |
```

Comment:
"...is designed to assist DevOps users in writing Terraform code and streamlining..."